### PR TITLE
PPTP-416 Add PPT landing page 

### PR DESCRIPTION
**Pull request by PauloPortugal**
**Work involved:**

 * Rename all messages to identify sections
 * Format 'main_template'. Easier to read
 * Add Feedback content to phaseBanner
 * Add PPT landing page
 * Increase code coverage to 93%
 * Integrate with the returns' journey
 * Put homepage behind auth

  The wave warning is due to a repetition of the URT. This will disappear once we develop the service

Screenshot from 2021-03-18 15-49-24

### CODE REVIEW

File: test/uk/gov/hmrc/plasticpackagingtax/returns/views/home/HomePageViewSpec.scala [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  private def createView(): Html = homePage()
+
+  "Home Page view" should {
+
+    "have proper messages for labels" in {
+
+      messages must haveTranslationFor("account.homePage.title")
+      messages must haveTranslationFor("account.homePage.currentBalance")
+      messages must haveTranslationFor("account.homePage.currentBalance")
+      messages must haveTranslationFor("account.homePage.currentBalance")
```
Comment (tmc-ee) [inline test/uk/gov/hmrc/plasticpackagingtax/returns/views/home/HomePageViewSpec.scala:45]: Should these be repeated here?

**Review by tmc-ee — APPROVED** [review]

File: app/uk/gov/hmrc/plasticpackagingtax/returns/views/home/home_page.scala.html [hunk 2]
```diff
@@ -14,21 +14,52 @@
...
 @()(implicit request: Request[_], messages: Messages)
 
-@govukLayout(title = Title("startPage.title")) {
+@govukLayout(title = Title("account.homePage.title"), useCustomContentWidth = true) {
+    @pageTitle(text = messages("account.homePage.title"), classes = gdsPageHeadingL)
 
-<div class="govuk-!-width-two-thirds">
-    @pageTitle(text = messages("startPage.title"))
-</div>
+    <h2 id="entityName" class="@gdsPageHeadingL">Plastic Packaging Ltd.</h2>
```
[the remark this answers is not in the document]
Comment (PauloPortugal) [reply app/uk/gov/hmrc/plasticpackagingtax/returns/views/home/home_page.scala.html:41]: It should come from EIS/ETMP, for the time being we are hardcoding it.

File: test/uk/gov/hmrc/plasticpackagingtax/returns/views/home/HomePageViewSpec.scala (hunk 1 above)
Comment (PauloPortugal) [reply test/uk/gov/hmrc/plasticpackagingtax/returns/views/home/HomePageViewSpec.scala:45]: Well spotted.

Comment (PauloPortugal) [reply test/uk/gov/hmrc/plasticpackagingtax/returns/views/home/HomePageViewSpec.scala]: Resolved!

**Review by tombart — APPROVED** [review]

**Pull request merged**